modules/deploy/pages/deployment-option/cloud/api/cloud-controlplane-api.adoc

```diff
@@ -0,0 +1,189 @@
+
+== Control Plane API
+
+The Control Plane API is one central API that allows you to provision clusters, networks, and resource groups.
+
+The Control Plane API consists of the following endpoints:
+
+* pass:a,m[xref:{tag-clusters}[ClusterService\]]
+* pass:a,m[xref:{tag-networks}[NetworkService\]]
+* pass:a,m[xref:{tag-operations}[OperationService\]]
+* pass:a,m[xref:{tag-resource-groups}[ResourceGroupService\]]
+
+== Long-running operations
+
+Some endpoints do not directly return the resource itself, but instead return an operation. The following is an example response of xref:api:ROOT:cloud-api.adoc#post-/v1beta2/networks[Create Network] (`POST /networks`):
+
+[,bash,role=no-copy]
+----
+"operation": {
+  "id": "cl5nlt1c1scg01plkoe0",
+  "metadata": {
+    "@type": "type.googleapis.com/redpanda.api.controlplane.v1beta2.CreateNetworkMetadata",
+    "network_id": "cl5o0td1dafe70i7460g"
+  },
+  "state": "STATE_IN_PROGRESS",
+  "started_at": "2023-11-08T12:35:01.957Z",
+  "finished_at": null
+}
+----
+
+The response object represents the long-running operation of creating a network. Network creation is an example of an operation that can take a longer period of time to complete.
+
+=== Check operation state
+
+To check the progress of an operation, make a request to the xref:api:ROOT:cloud-api.adoc#get-/v1beta2/operations/-id-[`GET /operations/\{id}`] endpoint using the operation ID as a parameter: 
+
+```bash
+curl https://api.redpanda.com/v1beta2/operations/<operation-id>
+```
+
+The response contains the current state of the operation: `IN_PROGRESS`, `COMPLETED`, or `FAILED`.
+
+== Throughput tiers
+
+When you create a Dedicated or BYOC cluster, you select a throughput tier. Throughput tiers define the sizing of a cluster. Availability depends on the region and the cluster type.
+
+- xref:deploy:deployment-option/cloud/dedicated/dedicated-tiers.adoc[Dedicated Cloud throughput tiers and regions]
+- xref:deploy:deployment-option/cloud/byoc-tiers.adoc[BYOC throughput tiers and regions]
+
+== Create a cluster
+
+To create a new cluster, first create a resource group and network if you have not already done so.
+
+=== Create a resource group 
+
+Create a resource group by making a POST request to the xref:api:ROOT:cloud-api.adoc#post-/v1beta2/resource-groups[`/v1beta2/resource-groups`] endpoint. Pass a name for your resource group in the request body.
+
+[,bash]
+----
+curl -H 'Content-Type: application/json' \
+-H 'Authorization: Bearer <token>' \
+-d '{
+  "name": "<resource-group-name>"
+}' -X POST https://api.redpanda.com/v1beta2/resource-groups
+----
+
+A resource group ID is returned. Pass this ID later when you call the Create Cluster endpoint.
+
+=== Create a network
+
+Create a network by making a request to xref:api:ROOT:cloud-api.adoc#post-/v1beta2/networks[`POST /v1beta2/networks`].
+
+Choose a CIDR range that does not overlap with your existing VPCs or your Redpanda network.
```

Comment:
You could consider adding a link for more info: https://deploy-preview-411--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/cidr-ranges/